template/src/{{module_name}}/napari.yaml.jinja

```
@@ -54,4 +54,10 @@ contributions:
       autogenerate: true
       display_name: Autogenerate Threshold
     - command: {{plugin_name}}.make_qwidget
-      display_name: Example QWidget{% endif %}
+      display_name: Example QWidget
+  menus:
+    napari/layers/filter:
```

```suggestion
    napari/layers/segment:
```
filter doesn't seem to be the correct menu since these are all thresholding widgets, but I know people think about thresholding different so 🤷